docs/ops.html

```diff
@@ -2250,57 +2174,19 @@ <h5><a id="kafka_streams_cache_monitoring" href="#kafka_streams_cache_monitoring
         <th>Mbean name</th>
       </tr>
       <tr>
-        <td>hitRatio-avg</td>
-        <td>The average cache hit ratio defined as the ratio of cache read hits over the total cache read requests. </td>
-        <td>kafka.streams:type=stream-record-cache-metrics,client-id=([-.\w]+),task-id=([-.\w]+),record-cache-id=([-.\w]+)</td>
-      </tr>
-      <tr>
-        <td>hitRatio-min</td>
-        <td>The mininum cache hit ratio. </td>
-        <td>kafka.streams:type=stream-record-cache-metrics,client-id=([-.\w]+),task-id=([-.\w]+),record-cache-id=([-.\w]+)</td>
-      </tr>
-      <tr>
-        <td>hitRatio-max</td>
-        <td>The maximum cache hit ratio. </td>
-        <td>kafka.streams:type=stream-record-cache-metrics,client-id=([-.\w]+),task-id=([-.\w]+),record-cache-id=([-.\w]+)</td>
-      </tr>
-    </tbody>
- </table>
-
-  <h5><a id="kafka_streams_buffer_monitoring" href="#kafka_streams_buffer_monitoring">Suppression Buffer Metrics</a></h5>
-  All the following metrics have a recording level of <code>debug</code>:
-
-  <table class="data-table">
-      <tbody>
-      <tr>
-        <td>suppression-buffer-size-current</td>
```

Comment:
Where did we document the deprecation of suppression related metrics?

Reply:
They were not deprecated but moved to state store metrics.